generator/templates/wire/WireCmd.cpp

```diff
@@ -176,10 +176,10 @@
                 size_t memberLength = {{member_length(member, "record.")}};
                 auto memberBuffer = reinterpret_cast<{{member_transfer_type(member)}}*>(buffer);
 
+                buffer += memberLength * {{member_transfer_sizeof(member)}};
```

Comment:
Nice catch, this must have been tricky to find.

Reply:
Took some time with a debugger (and noticing that `transfer` and `buffer` ended up being equal at some point). Unfortunately the test I wrote wasn't catching it (although now it is? confused about that.)